### ISSUE TITLE
Scale_Sector_Size throws error "wave_active"

### ISSUE THREAD

**Issue by MarcusInVR** [bvbohnen/X4_Customizer#5]
Unfortunately not completely compatible with Tides of Avarice - when attempting to rescale the game board without any extensions/mods installed, one will see the mentioned error.

Same for version in Releases, or when done from Visual Studio.

**Comment by nathansmith11170** [comment]
I also experience this issue

**Comment by nathansmith11170** [comment]
I think the problem is that the region definitions of the ego_dlc_pirate directory in the file ego_dlc_pirate\libraries\region_definitions.xml are not being loaded, and this file is not a diff so it doesn't get applied to the file in the root libraries directory. I could be wrong, though.

**Comment by nathansmith11170** [comment]
The other problem is that the 'wave_active' region has radius of None, from Update_Radius, but is linked with a connection unlike test regions, so the reference to the key 'wave_active' would throw KeyError even if the regions defined by ego_dlc_pirate are included by default.

I think the reason Update_Radius is returning null is that 'wave_active' has a <boundaries> tag before any <boundary> exists, but the XPath for Update_Radius is pointing at './boundary'

**Comment by nathansmith11170** [comment]
Yes, it appears that the schema for \<region\> now allows for \<boundaries\> after the Tides of Avarice DLC and the Region.py file does not account for this, since many of its xpath expressions rely on the existence of \<boundary\> as an immediate child tag of \<region\>

**Comment by SteelBlueVision** [comment]
Will this get fixed?
